config.default.yaml

```diff
@@ -261,6 +261,7 @@ clustering:
   cluster_network:
     algorithm: kmeans
     feature: solar+onwind-time
+  exclude_carriers: [] # list of carriers which will not be aggregated. If empty, all carriers will be aggregated.
```

Comment:
Please add documentation rather here:

https://github.com/PyPSA/pypsa-eur/blob/master/doc/configtables/clustering.csv

than directly in the `config.*.yaml` files, because there it is always duplicated.